weave/trace/op.py

```diff
@@ -234,7 +234,6 @@ def handle_exception(e: Exception) -> Any:
     if inspect.iscoroutinefunction(func):
 
         async def _call_async() -> Coroutine[Any, Any, Any]:
-            call_context.push_call(call)
```

Comment:
Is it correct to pop_call both here and in `finish_call`?

Reply:
oops yeah, deleted the extraneous pop